apps/daffio/src/app/core/sidebar/containers/sidebar/sidebar.component.ts

```diff
@@ -5,18 +5,18 @@ import { Component, Output, EventEmitter } from '@angular/core';
   templateUrl: './sidebar.component.html',
   styleUrls: ['./sidebar.component.scss']
 })
-export class DaffioSidebarContainer {
-
-  @Output() close: EventEmitter<any> = new EventEmitter();
-
-  onClose() {
-    this.close.emit();
-  }
+export class DaffioSidebarContainerComponent {
```

Comment:
I think this should be named `DaffioSidebarContainer`

Reply:
resolved in dbcf3deb24b3190d5c086b952e576b6e7fca991d